src/services/userAuthentication.js

```diff
@@ -28,22 +24,67 @@ class UserAuthentication {
         throw err
       } else {
         that.username=res['user'];
-        return that.findOrCreateUser(that.username);
+        return that.userInOurDb = that.findOrCreateUser(that.username);
       }
     });
   }
+  followSteemUser(steemUserNameToFollow) {
+    let that=this;
+    return this.steemUser.follow(
+      this.username,
+      steemUserNameToFollow,
+      function(err, res) {
+        console.log(err);
+        return err
+          ? new Error(err.error_description)
+          : that.userInOurDb;
+      }
+    );
+  }
+
+  unFollowSteemUser(steemUserNameToUnfollow) {
+    return this.steemUser.unfollow(
+      this.username,
+      steemUserNameToUnfollow,
+      function(err, res) {
+        return err
+          ? new Error(err.error_description)
+          : that.userInOurDb;
+      }
+    );
+  }
 
   findOrCreateUser(username) {
     return UserModel
       .findOrCreate({
-        where: {name: this.username },
+        where: {name: username },
         defaults: { id: _.random(10000)}
       })
       .spread((user, created) => {
         return user;
     });
   }
 
+  initializeSteemUser(accessToken) {
+    return this.steemUser = sc2.Initialize({
+      app: 'hundredx.app',
+      callbackURL: process.env.DEV_URL,
+      accessToken,
+    });
+  }
+
+  getUsersFollowerPosts() {
```

Comment:
This name is a bit confusing. Not sure what it's fetching even after reading the code haha

Reply:
how bout getPostsOfMyFollowers

Reply:
yeah that works